src/ontology/mondo-edit.obo

```diff
@@ -139282,9 +139286,10 @@ xref: UMLS:C1838416 {source="OMIM:119600", source="MONDO:notFoundInDiseaseSubset
 xref: UMLS:C1861516 {source="OMIM:119600", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0002254 {source="NCIT:C75020"} ! syndromic disease
 is_a: MONDO:0005516 {source="DOID:13994", source="MESH:D002973"} ! osteochondrodysplasia
-is_a: MONDO:0018230 {source="MONDO:0019709-obsoleted", source="https://github.com/monarch-initiative/mondo/pull/2571/"} ! skeletal dysplasia
+relationship: excluded_subClassOf MONDO:0018230 {source="MONDO:0019709-obsoleted", source="https://orcid.org/0000-0001-5208-3432"} ! skeletal dysplasia
```

Comment:
I would remove but not "exclude_SC"
The parent "osteochondrodysplasia" seems that is should be "skeletal dysplasia" (though it currently is not).